test-cases/longevity/longevity-mv-synchronous-updates-12h.yaml

```
@@ -0,0 +1,27 @@
+test_duration: 780
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=10000200  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(5)' -pop seq=1..10000200 -log interval=15",
```

```suggestion
prepare_write_cmd: ["cassandra-stress write cl=ALL n=104857600  -schema 'replication(factor=3)' -mode cql3 native -rate threads=1000 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..104857600 -log interval=15",
```

1. I don't see a reason to specifically use LCS.
2. Changed the size to 100GB instead of 50GB.
3. Changed to CL=ALL because you don't have nemesis during prepare.

---

changed